google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -41,9 +41,28 @@ void TracingMessageBatch::SaveMessage(pubsub::Message m) {
   child_->SaveMessage(std::move(m));
 }
 
+std::string ToString(opentelemetry::trace::TraceId trace_id) {
```

Comment:
Consider pass-by-value -v- pass-by-const ref, particularly given that the version in `opentelemetry_matchers.h` appears to make the opposite choice.  Typically we'd first determine whether the value is consumed, and then the relative costs of each choice.

Reply:
Done. 

The value is only read so past by ref sounds good.